docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trellix-mvisio-epo-source.md

```diff
@@ -0,0 +1,122 @@
+---
+id: trellix-mvisio-epo-source
+title: Trellix mVision ePO Source
+sidebar_label: Trellix mVision ePO
+description: Learn how to collect event logs using the Trellix mVision ePO.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/trellix-logo.png')} alt="trellix-logo" width="70" />
+
+Trellix is a cybersecurity company that provides cloud-based security solutions for cybersecurity attacks. It provides hardware, software, and services to investigate cybersecurity attacks, protect against malicious software, and analyze IT security risks.
+
+mVision ePO is a key component of the Trellix security management platform, which provides unified management of endpoint, network, and data security, which reduces incident response time, strengthen protection, simplify risk and security management with automation, and end-to-end network visibility and security.
```

Comment:
```suggestion
mVision ePO is a key component of the Trellix security management platform, which provides unified management of endpoint, network, and data security. This can reduce incident response time, strengthen protection, simplify and automate risk and security management, and provide end-to-end network visibility and security.
```